port/unix/omrsysinfo.c

```diff
@@ -2004,6 +2013,74 @@ omrsysinfo_get_OS_version(struct OMRPortLibrary *portLibrary)
 					rc = sysctlbyname(KERN_OSPRODUCTVERSION, buffer, &resultSize, NULL, 0);
 				}
 				buffer[resultSize] = '\0';
```

Comment:
`buffer` may refer to freed memory here.

Reply:
There's no `mem_free_memory` called between when `buffer` was declared in L1999 and here. I don't see how.

Reply:
Yes, you are correct. I read the code as if the reference was to the `buffer` declared on line 1940 (and possibly freed on line 1988).